spec/features/crops/crop_wrangling_button_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+feature "crop wrangling button" do
+
+  let(:crop_wrangler_member) { FactoryGirl.create(:crop_wrangler_member) }
+
+    context "crop wrangling button" do
```

Comment:
Good job using `context` to group tests, but probably only need to test two contexts. One when you're logged in as a wrangler and one when you're not. Testing the Admin member might be overkill (*in my opinion*).